fe/fe-core/src/main/java/org/apache/doris/nereids/trees/expressions/functions/scalar/Sha2.java

```diff
@@ -41,13 +44,31 @@ public class Sha2 extends ScalarFunction
             FunctionSignature.ret(VarcharType.SYSTEM_DEFAULT).args(VarcharType.SYSTEM_DEFAULT, IntegerType.INSTANCE),
             FunctionSignature.ret(VarcharType.SYSTEM_DEFAULT).args(StringType.INSTANCE, IntegerType.INSTANCE));
 
+    private static final List<Integer> validDigest = Lists.newArrayList(224, 256, 384, 512);
```

Comment:
static variable's name should use UPPER_CASE name format

Reply:
ok. will fix in next pr